windows/micropython.vcxproj

```diff
@@ -75,10 +76,25 @@
   <PropertyGroup Label="UserMacros">
     <CustomPropsFile Condition="'$(CustomPropsFile)'==''">msvc/user.props</CustomPropsFile>
   </PropertyGroup>
-  <PropertyGroup />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <OutDir>$(PyOutDir)$(Configuration)\$(Platform)\</OutDir>
+  </PropertyGroup>
```

Comment:
Yep, I didn't notice it was already defined in the props file. I am unfamiliar with the other ports but dumping the binaries in the same folder as the rest of the code sounds like a pretty bad idea to me: hard to do cleanup and confusing. I thought that building had failed because I couldn't find the binaries folder!